### PR TITLE
Upgrade terraform-provider-pagerduty to v2.9.3

### LINKED ISSUE #112: Upgrade terraform-provider-pagerduty to v2.6.2
Release details: https://github.com/PagerDuty/terraform-provider-pagerduty/releases/tag/v2.6.2

### LINKED ISSUE #117: Upgrade terraform-provider-pagerduty to v2.6.3
Release details: https://github.com/PagerDuty/terraform-provider-pagerduty/releases/tag/v2.6.3

### LINKED ISSUE #118: Upgrade terraform-provider-pagerduty to v2.6.4
Release details: https://github.com/PagerDuty/terraform-provider-pagerduty/releases/tag/v2.6.4

### LINKED ISSUE #120: Upgrade terraform-provider-pagerduty to v2.6.5
Release details: https://github.com/PagerDuty/terraform-provider-pagerduty/releases/tag/v2.6.5

### LINKED ISSUE #121: Upgrade terraform-provider-pagerduty to v2.7.0
Release details: https://github.com/PagerDuty/terraform-provider-pagerduty/releases/tag/v2.7.0

### LINKED ISSUE #122: Upgrade terraform-provider-pagerduty to v2.8.0
Release details: https://github.com/PagerDuty/terraform-provider-pagerduty/releases/tag/v2.8.0

### LINKED ISSUE #123: Upgrade terraform-provider-pagerduty to v2.8.1
Release details: https://github.com/PagerDuty/terraform-provider-pagerduty/releases/tag/v2.8.1

### LINKED ISSUE #124: Upgrade terraform-provider-pagerduty to v2.9.0
Release details: https://github.com/PagerDuty/terraform-provider-pagerduty/releases/tag/v2.9.0

### LINKED ISSUE #125: Upgrade terraform-provider-pagerduty to v2.9.1
Release details: https://github.com/PagerDuty/terraform-provider-pagerduty/releases/tag/v2.9.1

### LINKED ISSUE #129: Upgrade terraform-provider-pagerduty to v2.9.2
Release details: https://github.com/PagerDuty/terraform-provider-pagerduty/releases/tag/v2.9.2

### LINKED ISSUE #130: Upgrade terraform-provider-pagerduty to v2.9.3
Release details: https://github.com/PagerDuty/terraform-provider-pagerduty/releases/tag/v2.9.3

### PR DESCRIPTION
**Pull request by aq17**
Fixes #130 
Fixes #129 
Fixes #125 
Fixes #124 
Fixes #123 
Fixes #122 
Fixes #121 
Fixes #120 
Fixes #118 
Fixes #117 
Fixes #112

### CODE REVIEW

File: provider/resources.go [hunk 1]
```diff
@@ -104,9 +119,11 @@ func Provider() tfbridge.ProviderInfo {
...
 				},
 			},
-			"pagerduty_event_rule":           {Tok: makeResource(mainMod, "EventRule")},
-			"pagerduty_extension":            {Tok: makeResource(mainMod, "Extension")},
-			"pagerduty_extension_servicenow": {Tok: makeResource(mainMod, "ExtensionServiceNow")},
+			"pagerduty_event_rule":                {Tok: makeResource(mainMod, "EventRule")},
+			"pagerduty_extension":                 {Tok: makeResource(mainMod, "Extension")},
+			"pagerduty_extension_servicenow":      {Tok: makeResource(mainMod, "ExtensionServiceNow")},
+			"pagerduty_incident_workflow":         {Tok: makeResource(mainMod, "PagerdutyIncidentWorkflow")},
+			"pagerduty_incident_workflow_trigger": {Tok: makeResource(mainMod, "PagerdutyIncidentWorkflowTrigger")},
```
Comment (iwahbe) [inline provider/resources.go:126]: Why do these resources have a "Pagerduty" prefix?

```suggestion
			"pagerduty_incident_workflow":         {Tok: makeResource(mainMod, "IncidentWorkflow")},
			"pagerduty_incident_workflow_trigger": {Tok: makeResource(mainMod, "IncidentWorkflowTrigger")},
```

**Review by iwahbe — CHANGES_REQUESTED** [review]

**Comment by iwahbe** [comment]
It's a lot easier to review these changes when you use the auto-mapper introduced in `3.38.1`.

**Comment by aq17** [comment]
> It's a lot easier to review these changes when you use the auto-mapper introduced in `3.38.1`.

👍🏼 I'll refactor to do so.
The sdks are building successfully locally, not quite sure why they are failing here

**Comment by AaronFriel** [comment]
@aq17 are files left over from the previous commit? Deleting the sdks and rebuilding tends to fix that.

**Comment by t0yv0** [comment]
Something still odd with the Go SDK builds.

**Review by AaronFriel — COMMENTED** [review]
I misspoke earlier, looks like we need to keep these files as tfgen didn't create them.

**Comment by aq17** [comment]
Closing in favor of https://github.com/pulumi/pulumi-pagerduty/pull/136.

**Pull request closed without merging**